tests/integration/plugins/test_poetry/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "test-poetry"
+version = "0.1.0"
+description = ""
+authors = ["Alex Lowe <alex.lowe@canonical.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
```

Comment:
suggestion: add another dependency (eg black or requests) and use it in the hello world (e.g. print black's version)
this is to check that the dependency was installed and can be found in the virtual env correctly

Reply:
Good call! I added an older version of `distro` (since it doesn't have second-order dependencies) and checked the version in the script.